pkg/csi/service/vanilla/controller.go

```diff
@@ -1804,28 +2062,36 @@ func (c *controller) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequ
 				"volume capability not supported. Err: %+v", err)
 		}
 		if common.IsFileVolumeRequest(ctx, volumeCapabilities) {
-			// Error out if TopologyRequirement is provided during file volume provisioning
+			// Error out if TopologyRequirement is provvanilla/controller.go:1884ided during file volume provisioning
 			// as this is not supported yet.
-			if req.GetAccessibilityRequirements() != nil {
-				return nil, csifault.CSIInvalidArgumentFault, logger.LogNewErrorCode(log, codes.InvalidArgument,
-					"volume topology feature for file volumes is not supported.")
-			}
+			// if req.GetAccessibilityRequirements() != nil {
```

Comment:
put this in a FSS check instead of removing it

Reply:
Done